### PR TITLE
161348756 Users should be able to login via Google or Facebook

### PR DESCRIPTION
**Pull request by RachaelNantale**
#### What does this PR do?
Provides buttons or icons where a user can log in via social media
#### Description of Task to be completed?
 - A user is able to login via Google
 - A user is able to login via Facebook

#### How should this be manually tested?
- Go to `/login`
- click on either `facebook` or `Google`
#### Any background context you want to provide?
You need to have signed up before logging in
#### What are the relevant pivotal tracker stories?
[# 161348756](https://www.pivotaltracker.com/story/show/161348756)

### CODE REVIEW

File: src/commons/axiosInstance.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import axios from 'axios';
```
Comment (brucemakallan) [inline src/commons/axiosInstance.js:1]: Thank you for separating this concern so that the team can just call it later.

**Review by brucemakallan — DISMISSED** [review]

File: src/commons/axiosInstance.js (hunk 1 above)
Comment (brucemakallan) [reply src/commons/axiosInstance.js]: I checked it out on Heroku review apps and we'll need to test out facebook probably after the App is no longer in development mode.

File: src/components/Login/index.js [hunk 2]
```diff
@@ -3,12 +3,13 @@ import * as PropTypes from 'prop-types';
 import './Login.scss';
 import { library } from '@fortawesome/fontawesome-svg-core';
 import { fab } from '@fortawesome/free-brands-svg-icons';
-import SocialMediaLinks from '../SocialMediaLinks';
+// import SocialMediaLinks from '../SocialMediaLinks';
```
Comment (judeinno) [inline src/components/Login/index.js:6]: Please remove commented out code

File: src/components/Login/socialLogin.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { Redirect } from 'react-router-dom';
+import { connect } from 'react-redux';
+// import FacebookLogin from 'react-facebook-login';
```
Comment (judeinno) [inline src/components/Login/socialLogin.js]: Remove this too

**Review by brucemakallan — DISMISSED** [review]

**Comment by dnuwa** [comment]
@RachaelNantale kindly add some images to your pr description.

File: yarn-error.log [hunk 4]
```diff
@@ -0,0 +1,150 @@
+Arguments:
```
Comment (tibetegya) [inline yarn-error.log]: remove this file "yarn-error.log"

File: src/reducers/tests/reducers.test.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
+import expect from 'expect';
+import combinedReducers from '../index';
+import userReducer from '../userReducer';
+import articleReducer from '../articleReducer';
+import ACTION_TYPES from '../../actions/actionTypes';
+import initialState from '../../commons/initialStates';
+
+describe('post reducer', () => {
+  it('should returns the initial state', () => {
+    expect(combinedReducers(undefined, {})).toEqual(initialState);
```
Comment (tibetegya) [inline src/reducers/tests/reducers.test.js:10]: why are you passing undefined into this test?

**Review by judeinno — DISMISSED** [review]

File: src/reducers/tests/reducers.test.js (hunk 5 above)
Comment (RachaelNantale) [reply src/reducers/tests/reducers.test.js:10]: I am passing the state as undefined so that it can bring back the initial state.

**Pull request merged**